### ISSUE TITLE
Data import error [Unnamed2344]

### ISSUE THREAD

**Issue by tashrifbillah** [redcap-tools/PyCap#151]
I am importing records as rows from the web interface:

![image](https://user-images.githubusercontent.com/35086881/121436393-7dcab200-c94e-11eb-81e0-ca18d293df27.png)

After uploading data, it shows the following error
![image](https://user-images.githubusercontent.com/35086881/121436410-8622ed00-c94e-11eb-92bf-ecf6f78bff24.png)

---

I did a Ctrl+F on my codebook webpage but could not find any such variable. So, what line of my data dictionary is causing this error?

> The following fields were not found in the project as real data fields: Unnamed2344

New to REDCap programming, cut me some slack :) :)

**Comment by tashrifbillah** [comment]
Hi @JuliaSprenger , requesting your input based on your contribution to similar work in this repo.

**Comment by JuliaSprenger** [comment]
Hi @tashrifbillah,
this is hard to tell, as 1) it's not PyCap related but an issue encountered in the RedCap webinterface and 2) you don't provide the file you are trying to upload. Have you tried uploading reduced versions of your csv file to localize the problematic entry?
Also I think you should contact the RedCap Admin at your institute who can help you more directly for questions like this.

**Comment by tashrifbillah** [comment]
Thank you for your comment. I have found the issue. The `data import template` that I downloaded from REDCap came with an unnamed last column! I guess REDCap appended an empty row to my *data dictionary* which got transposed to the empty unnamed column.

To solve this, after downloading the `data import template`, I have deleted that unnamed column from within an excel viewer. The error seems to be gone now.
